### PR TITLE
Update Java q and r example to use N2 instead of 2

### PR DESCRIPTION
**Pull request by mcm001**
The previous code did not compile. This addresses that error.

### CODE REVIEW

**Comment by calcmogul** [comment]
To be clear, that code is being updated so it actually compiles.

File: source/docs/software/advanced-controls/state-space/state-space-intro.rst [hunk 1]
```diff
@@ -167,23 +167,28 @@ For example, we might use the following Q and R for an elevator system with posi
...
 
-         #include <array>
-
-         // q's elements
-         std::array<double, 2> qElms{0.1, 0.5);
-
-         // r's elements
-         std::array<double, 1> rElms{12.0};
+          LinearSystem<2, 1, 1> elevatorSystem = <our elevator>;
+          LinearQuadraticRegulator<2, 1> controller{
```
Comment (prateekma) [inline source/docs/software/advanced-controls/state-space/state-space-intro.rst:184]: Why are the elements different between Java and C++?

**Review by prateekma — CHANGES_REQUESTED** [review]

File: source/docs/software/advanced-controls/state-space/state-space-intro.rst [hunk 2]
```diff
@@ -167,23 +167,28 @@ For example, we might use the following Q and R for an elevator system with posi
 
       .. code-block:: Java
 
-         // q's elements
-         Vector<2> qElms = VecBuilder.fill(0.1, 0.5);
-
-         // r's elements
-         Vector<1> rElms = VecBuilder.fill(12.0);
+         LinearSystem<N2, N1, N1> elevatorSystem = <our elevator>;
+         LinearQuadtraticRegulator<N2, N1, N1> controller = new LinearQuadraticRegulator(system,
```
Comment (calcmogul) [inline source/docs/software/advanced-controls/state-space/state-space-intro.rst:171]: This still doesn't compile.

File: source/docs/software/advanced-controls/state-space/state-space-intro.rst [hunk 3]
```diff
@@ -167,23 +167,30 @@ For example, we might use the following Q and R for an elevator system with posi
...
       .. code-block:: C++
 
-         #include <array>
-
-         // q's elements
-         std::array<double, 2> qElms{0.1, 0.5);
-
-         // r's elements
-         std::array<double, 1> rElms{12.0};
+         // Example system -- must be changed to match your robot.
```
Comment (calcmogul) [inline source/docs/software/advanced-controls/state-space/state-space-intro.rst:184]: The comment indentation is off by one space.

**Review by calcmogul — APPROVED** [review]

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**